Assets/Scenes/Main.unity

```diff
@@ -3196,7 +3196,7 @@ MonoBehaviour:
   m_SdkMode: 1
   m_AutoProfile: 0
   m_AutoProfileWaitTime: 10
-  Secrets: {fileID: 11400000, guid: 2f5f8e93c6bc7be49a5023148b50e7a0, type: 2}
+  Secrets: {fileID: 11400000, guid: 94ce910a09cb4944f9100b4be96da57f, type: 2}
```

Comment:
What was this change about? From the commit, I thought it might have been undoing something in the branch, but I see it's still present.

Reply:
Please revert before merge